tools/mqppep/mqppep_preproc.xml

```diff
@@ -0,0 +1,520 @@
+<tool id="mqppep_preproc" name="MaxQuant Phosphopeptide Preprocessing" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5">
+    <description>
+        Transform the output of MaxQuant for phosphoproteome-enriched samples to prepare it for statistical anlaysis.
+    </description>
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+      (
+        echo '--- localization-filter step:'
+      ) && (
+        \$CONDA_PREFIX/bin/Rscript '$__tool_directory__/MaxQuantProcessingScript.R'
```

Comment:
```suggestion
        Rscript '$__tool_directory__/MaxQuantProcessingScript.R'
```

Reply:
@bgruening My tool **does not run** in our production Galaxy without the `CONDA_PREFIX`.

My tool **does run** with `planemo test --biocontainers .`

Reply:
@bgruening After I added  `profile="21.05"`, this ran fine in our production Galaxy.  I don't know whether that was the cause of the success, but I'll take it!

Implemented in commit 3d51a38113ad440ac4b8e6e569d9f6cae936cf7a